Manual/Releases.lean

```diff
@@ -0,0 +1,35 @@
+/-
+Copyright (c) 2025 Lean FRO LLC. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Author: Joachim Breitner
+-/
+
+import VersoManual
+
+import Manual.Releases.«v4.15.0»
+import Manual.Releases.«v4.16.0»
+import Manual.Releases.«v4.17.0»
+
+open Manual
+open Verso.Genre
+
+#doc (Manual) "Release Notes" =>
+%%%
+tag := "release-notes"
+file := "releases"
+number := false
+%%%
+
+This section provides release notes about recent versions of Lean and is presented in reverse
+chronological order. When updating your copy of Lean to a new version, it is strongly recommended
+that you read the corresponding release notes. They may contain advice that will help you understand
+the differences with the previous version and upgrade your projects.
```

Comment:
```suggestion
This section provides release notes about recent versions of Lean. When updating to a new version, please
read the corresponding release notes. They may contain advice that will help you understand
the differences with the previous version and upgrade your projects.
```

NItpicky rationales:
 * Readers can see that it's reverse order, I'm not sure it contributes much.
 * The user's copy of Lean doesn't get updated, really - the new version ends up next to it. This is admittedly nitpicky :)
 * "It is strongly recommended that" - this is a use of the passive voice that I think makes it harder to follow. I rephrased it from an existential claim that a recommendation exists into just being the recommendation :-)

Reply:
I basically just copied https://rocq-prover.org/doc/V8.20.0/refman/changes.html :-)